### PR TITLE
Add support for Django Markup (with example)

### PR DESCRIPTION
**Pull request by rafaelpaixao**

### CODE REVIEW

File: apyb/website/views.py [hunk 1]
```diff
@@ -20,3 +21,12 @@ def get_context_data(self, **kwargs):
         context['hero_photos'] = HERO_PHOTOS
 
         return context
+
+class About(TemplateView):
+    template_name = 'website/markdown.html'
+
+    def get_context_data(self, **kwargs):
+        context = super(About, self).get_context_data(**kwargs)
+        markdown_content = open("website/markdown/about.md", 'r').read()
```
Comment (jbiason) [inline apyb/website/views.py]: That's a cool solution: Although I don't believe we'll use direct files in the future, it paves the way to make it simpler to load from a database, for example.

Nice work :+1:

File: markup.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django_markup.markup import formatter
```
Comment (jbiason) [inline markup.py]: This file is not being used, right?

File: apyb/website/views.py (hunk 1 above)
Comment (pedrokiefer) [reply apyb/website/views.py]: This way you are reprocessing something that is mainly static.

Loading from a database is better, but even better would be to format the markdown content only once and serve it as a static file.

Comment (jbiason) [reply apyb/website/views.py]: Yup. But, again, it proves that we can easily use Markdown for the pages.

One problem I see about creating the static pages is that, unless we use something ugly like iframes (well, _I_ think they are ugly, maybe there is something more direct, like server side imports), we need to generate the pages in the same style as the main page; if, for some reason, we update the style, all the pages should be regenerated. It seems we would just add complexity for something simple.

Another solution is still generate the page every time, but add a CDN in front of the site (disclaimer: I work at a CDN company :stuck_out_tongue: )

Comment (jbiason) [reply apyb/website/views.py]: But, again: Not saying you're wrong -- that's actually the most perfect solution we could use -- it's just that rendering from database, which this PR proves it's possible, it's waaay simpler and has less complexities.

We could look at the full render as an improvement for this in the future. Maybe we should open an issue to investigate and point how to do it. What do you think?

Comment (pedrokiefer) [reply apyb/website/views.py]: From a performance point of view, the ideal solution would be to do both, cache it on a CDN, and regenerate whenever the style / content got updated.

(disclaimer: that's how we do it @ globo.com)

**Comment by jbiason** [comment]
Cool. For now, that we only have one page, it will work fine. When we have more pages, we can extend the current model to say which ones will appear in the main page and/or the page ordering and add a "list of pages" page to list everything. Even with that, the only thing we need to change is the URL and add the new view.

So, yeah, nice work!

**Pull request merged**